### PR TITLE
install: Add initial hardware configuration for navigator 

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: install/install.sh [hunk 1]
```diff
@@ -9,11 +9,47 @@ REMOTE="$REMOTE/$VERSION"
...
+    echo "Companion only supports ARM computers."
+    exit 1
+)
 
 # Check if the script is running as root
 [[ $EUID != 0 ]] && echo "Script must run as root."  && exit 1
 
+# Running in a BCM27* or BCM28, do necessary Raspberry configuration
+cat /proc/cpuinfo | grep -Eq "BCM(27|28)" && (
+    echo "Configuring board.."
```
Comment (Williangalvani) [inline install/install.sh:22]:
```suggestion
    echo "Raspberry Pi detected, configuring board.."
```

File: install/install.sh [hunk 2]
```diff
@@ -9,11 +9,47 @@ REMOTE="$REMOTE/$VERSION"
...
+
+    # Check for valid modules file to load kernel modules
+    if [ -f "/etc/modules" ]; then
+        MODULES_FILE="/etc/modules"
+    else
+        MODULES_FILE="/etc/modules-load.d/companion.conf"
+        touch "$MODULES_FILE" || true # Create if it does not exist
+    fi
+
+    echo "- Load kernel modules."
```
Comment (Williangalvani) [inline install/install.sh:41]:
```suggestion
    echo "- Set up kernel modules."
```
They will only be loaded after the reboot, right?

**Review by Williangalvani — APPROVED** [review]
looks good. I'm guessing x86 support (without navigator) will come in another pr?

File: install/install.sh (hunk 1 above)
Comment (patrickelectric) [reply install/install.sh:22]: This should work with any BCM27 and 28 SBC, not necessary raspberry.

Comment (Williangalvani) [reply install/install.sh:22]: anyway, I think we should tell the user which configuration is being applied

Comment (Williangalvani) [reply install/install.sh:22]: well, maybe in the next pr.

Comment (patrickelectric) [reply install/install.sh:22]: > anyway, I think we should tell the user which configuration is being applied

Check the following lines.

**Pull request merged**